### ISSUE TITLE
Could not log "request.graphql" event. NoMethodError: undefined method `compact_blank' for true

### ISSUE THREAD

**Issue by mariowise** [virtualshield/rails-graphql#38]
I just installed the gem into a brand new rails generated project (API type). I'm able to run queries but I'm getting this error printed into my logs on every request.

```
Could not log "request.graphql" event. NoMethodError: undefined method `compact_blank' for true [
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/log_subscriber.rb:181:in `mode_from'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/log_subscriber.rb:175:in `color'", 
"/usr/local/bundle/gems/rails-graphql-1.0.3/lib/rails/graphql/railties/log_subscriber.rb:101:in `header'", 
"/usr/local/bundle/gems/rails-graphql-1.0.3/lib/rails/graphql/railties/log_subscriber.rb:32:in `request'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/subscriber.rb:138:in `call'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/log_subscriber.rb:147:in `call'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications/fanout.rb:178:in `block in finish'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications/fanout.rb:25:in `block in iterate_guarding_exceptions'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications/fanout.rb:24:in `each'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications/fanout.rb:24:in `iterate_guarding_exceptions'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications/fanout.rb:122:in `each'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications/fanout.rb:177:in `finish'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications/fanout.rb:259:in `block in finish_with_values'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications/fanout.rb:25:in `block in iterate_guarding_exceptions'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications/fanout.rb:24:in `each'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications/fanout.rb:24:in `iterate_guarding_exceptions'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications/fanout.rb:258:in `finish_with_values'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications/fanout.rb:251:in `finish'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications/instrumenter.rb:64:in `instrument'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications.rb:210:in `instrument'", 
"/usr/local/bundle/gems/rails-graphql-1.0.3/lib/rails/graphql/request.rb:502:in `log_execution'", 
"/usr/local/bundle/gems/rails-graphql-1.0.3/lib/rails/graphql/request.rb:424:in `execute!'", 
"/usr/local/bundle/gems/rails-graphql-1.0.3/lib/rails/graphql/request.rb:147:in `execute'", 
"/usr/local/bundle/gems/rails-graphql-1.0.3/lib/rails/graphql/request.rb:83:in `execute'", 
"/usr/local/bundle/gems/rails-graphql-1.0.3/lib/rails/graphql/railties/controller.rb:72:in `gql_request'", 
"/usr/local/bundle/gems/rails-graphql-1.0.3/lib/rails/graphql/railties/controller.rb:58:in `gql_request_response'", 
"/usr/local/bundle/gems/rails-graphql-1.0.3/lib/rails/graphql/railties/controller.rb:32:in `execute'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_controller/metal/basic_implicit_render.rb:8:in `send_action'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/abstract_controller/base.rb:226:in `process_action'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_controller/metal/rendering.rb:193:in `process_action'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/abstract_controller/callbacks.rb:261:in `block in process_action'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/callbacks.rb:121:in `block in run_callbacks'", 
"/usr/local/bundle/gems/actiontext-7.2.1/lib/action_text/rendering.rb:25:in `with_renderer'", 
"/usr/local/bundle/gems/actiontext-7.2.1/lib/action_text/engine.rb:71:in `block (4 levels) in <class:Engine>'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/callbacks.rb:130:in `instance_exec'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/callbacks.rb:130:in `block in run_callbacks'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/callbacks.rb:141:in `run_callbacks'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/abstract_controller/callbacks.rb:260:in `process_action'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_controller/metal/rescue.rb:27:in `process_action'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_controller/metal/instrumentation.rb:77:in `block in process_action'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications.rb:210:in `block in instrument'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications/instrumenter.rb:58:in `instrument'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/notifications.rb:210:in `instrument'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_controller/metal/instrumentation.rb:76:in `process_action'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_controller/metal/params_wrapper.rb:259:in `process_action'", 
"/usr/local/bundle/gems/activerecord-7.2.1/lib/active_record/railties/controller_runtime.rb:39:in `process_action'", 
"/usr/local/bundle/gems/rails-graphql-1.0.3/lib/rails/graphql/railties/controller_runtime.rb:29:in `process_action'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/abstract_controller/base.rb:163:in `process'", 
"/usr/local/bundle/gems/actionview-7.2.1/lib/action_view/rendering.rb:40:in `process'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_controller/metal.rb:252:in `dispatch'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_controller/metal.rb:335:in `dispatch'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/routing/route_set.rb:67:in `dispatch'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/routing/route_set.rb:50:in `serve'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/journey/router.rb:53:in `block in serve'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/journey/router.rb:133:in `block in find_routes'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/journey/router.rb:126:in `each'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/journey/router.rb:126:in `find_routes'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/journey/router.rb:34:in `serve'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/routing/route_set.rb:896:in `call'", 
"/usr/local/bundle/gems/rack-3.1.8/lib/rack/etag.rb:29:in `call'", 
"/usr/local/bundle/gems/rack-3.1.8/lib/rack/conditional_get.rb:43:in `call'", 
"/usr/local/bundle/gems/rack-3.1.8/lib/rack/head.rb:15:in `call'", 
"/usr/local/bundle/gems/activerecord-7.2.1/lib/active_record/migration.rb:674:in `call'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/middleware/callbacks.rb:31:in `block in call'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/callbacks.rb:101:in `run_callbacks'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/middleware/callbacks.rb:30:in `call'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/middleware/executor.rb:16:in `call'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/middleware/debug_exceptions.rb:31:in `call'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/middleware/show_exceptions.rb:32:in `call'", 
"/usr/local/bundle/gems/railties-7.2.1/lib/rails/rack/logger.rb:41:in `call_app'", 
"/usr/local/bundle/gems/railties-7.2.1/lib/rails/rack/logger.rb:29:in `call'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/middleware/remote_ip.rb:96:in `call'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/middleware/request_id.rb:33:in `call'", 
"/usr/local/bundle/gems/rack-3.1.8/lib/rack/runtime.rb:24:in `call'", 
"/usr/local/bundle/gems/activesupport-7.2.1/lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/middleware/server_timing.rb:61:in `block in call'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/middleware/server_timing.rb:26:in `collect_events'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/middleware/server_timing.rb:60:in `call'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/middleware/executor.rb:16:in `call'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/middleware/static.rb:27:in `call'", 
"/usr/local/bundle/gems/rack-3.1.8/lib/rack/sendfile.rb:114:in `call'", 
"/usr/local/bundle/gems/actionpack-7.2.1/lib/action_dispatch/middleware/host_authorization.rb:143:in `call'", 
"/usr/local/bundle/gems/railties-7.2.1/lib/rails/engine.rb:535:in `call'", 
"/usr/local/bundle/gems/puma-6.4.3/lib/puma/configuration.rb:272:in `call'", 
"/usr/local/bundle/gems/puma-6.4.3/lib/puma/request.rb:100:in `block in handle_request'", 
"/usr/local/bundle/gems/puma-6.4.3/lib/puma/thread_pool.rb:378:in `with_force_shutdown'", 
"/usr/local/bundle/gems/puma-6.4.3/lib/puma/request.rb:99:in `handle_request'", 
"/usr/local/bundle/gems/puma-6.4.3/lib/puma/server.rb:464:in `process_client'", 
"/usr/local/bundle/gems/puma-6.4.3/lib/puma/server.rb:245:in `block in run'", 
"/usr/local/bundle/gems/puma-6.4.3/lib/puma/thread_pool.rb:155:in `block in spawn_thread'
```

**Comment by crashtech** [comment]
Can you check if [rails-71-fix](https://github.com/virtualshield/rails-graphql/tree/rails-71-fix) branch fixes your problem? I just need to fix Ruby 3.3 tests to release a new version.

**Comment by crashtech** [comment]
The fix is now available on v1.0.4.
